components/slider/slider.component.ts

```diff
@@ -103,6 +103,7 @@ import { NzExtendedMark, NzMarks, NzSliderHandler, NzSliderShowTooltip, NzSlider
         [tooltipVisible]="nzTooltipVisible"
         [tooltipPlacement]="nzTooltipPlacement"
         [dir]="dir"
+        (keydown)="onHandleKeyDown(handleIndex)"
```

Comment:
```suggestion
        (focusin)="onHandleKeyDown(handleIndex)"
```
Is it possible to use focusin here?

Reply:
updated, pls review it again~